charts/datadog/values.yaml

```diff
@@ -510,6 +510,15 @@ datadog:
     iast:
       # datadog.asm.iast.enabled -- Enable Application Security Management Interactive Application Security Testing by injecting `DD_IAST_ENABLED=true` environment variable to all pods in the cluster
       enabled: false
+  ## OTel collector related configuration
+  otelcollector:
+    enabled: false
+    ports:
```

Comment:
missing comment on the `datadog.otelcollector.enabled` and `datadog.otelcollector.ports`

also I'm wondering that it makes sens to add a name for each port as we have discussed.